shell/common/options_switches.h

```diff
@@ -5,127 +5,289 @@
-extern const char kRoundedCorners[];
-extern const char ktitleBarOverlay[];
-extern const char kOverlayButtonColor[];
-extern const char kOverlaySymbolColor[];
-extern const char kOverlayHeight[];
-
-// WebPreferences.
-extern const char kZoomFactor[];
-extern const char kPreloadScript[];
-extern const char kNodeIntegration[];
-extern const char kContextIsolation[];
-extern const char kExperimentalFeatures[];
-extern const char kScrollBounce[];
-extern const char kEnableBlinkFeatures[];
-extern const char kDisableBlinkFeatures[];
-extern const char kNodeIntegrationInWorker[];
-extern const char kWebviewTag[];
-extern const char kCustomArgs[];
-extern const char kPlugins[];
-extern const char kSandbox[];
-extern const char kWebSecurity[];
-extern const char kAllowRunningInsecureContent[];
-extern const char kOffscreen[];
-extern const char kUseSharedTexture[];
-extern const char kNodeIntegrationInSubFrames[];
-extern const char kDisableHtmlFullscreenWindowResize[];
-extern const char kJavaScript[];
-extern const char kImages[];
-extern const char kTextAreasAreResizable[];
-extern const char kWebGL[];
-extern const char kNavigateOnDragDrop[];
-extern const char kEnablePreferredSizeMode[];
-
-extern const char kHiddenPage[];
-
-#if BUILDFLAG(ENABLE_BUILTIN_SPELLCHECKER)
-extern const char kSpellcheck[];
-#endif
+inline constexpr std::string_view kTitle = "title";
+inline constexpr std::string_view kIcon = "icon";
+inline constexpr std::string_view kFrame = "frame";
+inline constexpr std::string_view kShow = "show";
+inline constexpr std::string_view kCenter = "center";
+inline constexpr std::string_view kX = "x";
+inline constexpr std::string_view kY = "y";
+inline constexpr std::string_view kWidth = "width";
+inline constexpr std::string_view kHeight = "height";
+inline constexpr std::string_view kMinWidth = "minWidth";
+inline constexpr std::string_view kMinHeight = "minHeight";
+inline constexpr std::string_view kMaxWidth = "maxWidth";
+inline constexpr std::string_view kMaxHeight = "maxHeight";
+inline constexpr std::string_view kResizable = "resizable";
+inline constexpr std::string_view kMovable = "movable";
+inline constexpr std::string_view kMinimizable = "minimizable";
+inline constexpr std::string_view kMaximizable = "maximizable";
+inline constexpr std::string_view kFullScreenable = "fullscreenable";
+inline constexpr std::string_view kClosable = "closable";
+
+// whether to keep the window out of mission control
+inline constexpr std::string_view kHiddenInMissionControl =
+    "hiddenInMissionControl";
+
+inline constexpr std::string_view kFullscreen = "fullscreen";
+
+// Whether the window should show in taskbar.
+inline constexpr std::string_view kSkipTaskbar = "skipTaskbar";
+
+// Start with the kiosk mode, see Opera's page for description:
+// http://www.opera.com/support/mastering/kiosk/
+inline constexpr std::string_view kKiosk = "kiosk";
+
+inline constexpr std::string_view kSimpleFullScreen = "simpleFullscreen";
```

Comment:
Is it possible to have the casing of kFullscreen and kSimpleFullScreen (and their subsequently set strings) be more in line with one another? So either kFullscreen and kSimpleFullscreen, or kFullScreen and kSimpleFullScreen.